src/clj/rems/db/test_data.clj

```diff
@@ -1129,7 +1164,36 @@
                              :resource-id res-organization-owner
                              :form-id form-organization-owner
                              :organization {:organization/id "organization1"}
-                             :workflow-id (:organization-owner workflows)})))
+                             :workflow-id (:organization-owner workflows)})
+
+    (let [applicant (users :applicant1)
+          handler (users :approver2)
+          reviewer (users :reviewer)
+          catid-1 (create-catalogue-item! {:actor owner
+                                           :title {:en "Test workflow"
+                                                   :fi "Testityövuo"
+                                                   :sv "Test arbetsflöde"}
+                                           :resource-id res1
+                                           :form-id form-with-public-and-private-fields
+                                           :organization {:organization/id "nbn"}
+                                           :workflow-id (:default workflows)})
+          catid-2 (create-catalogue-item! {:actor owner
+                                           :title {:en "Test workflow with private form"
+                                                   :fi "Testityövuo yksityisellä lomakkeella"
+                                                   :sv "Test arbetsflöde med privat blankett"}
+                                           :resource-id res2
+                                           :form-id form-private-nbn
+                                           :organization {:organization/id "nbn"}
+                                           :workflow-id (:default workflows)})
+          app-id (create-draft! applicant [catid-1 catid-2] "two-form draft application")]
+      (command! {:type :application.command/submit
+                 :application-id app-id
+                 :actor applicant})
+      (command! {:type :application.command/request-review
+                 :application-id app-id
+                 :actor handler
+                 :reviewers [reviewer]
+                 :comment "please have a look"}))))
```

Comment:
this allows demoing the feature a little easier by having an application (by alice) with handler (hannah) and reviewer (carl) that contains:
- catalogue item 1, with public/private fields in form
- catalogue item 2, with only private fields in form

alice & hannah should see all fields in application and pdf, while carl should see only public fields in either